packages/shared/src/variants/parallel.ts

```diff
@@ -68,9 +72,27 @@ export class ParallelGo extends AbstractGame<
       }
     }
 
-    this.staged[player] = move;
+    if (move === 'resign' || move === 'timeout') {
+      this.playerParticipation[player].dropOutAtRound = this.numberOfRounds;
 
-    if (Object.entries(this.staged).length !== this.numPlayers()) {
+      if (this.nextToPlay().length < 2) {
+        this.phase = 'gameover';
+        // TODO: declare winner
```

Comment:
I think this would suffice:

```
this.result = `Player ${this.nextToPlay()[0]} wins!`;
```

Reply:
Can't parallel go have more than two players?

Reply:
Yes, but we are in a block where the condition is `this.nextToPlay().length < 2` - the winner is the last one remaining right?

Reply:
Oh, that's right :smile:

Reply:
right, only I wasn't sure if nextToPlay might return an empty array, so I ignored it for now.

Reply:
fixed